addons/payment_stripe/views/stripe.xml

```diff
@@ -4,6 +4,8 @@
         <template id="stripe_acquirer_button">
             <form t-if="acquirer" method="post" target="_self">
                 <script src="https://checkout.stripe.com/checkout.js"></script>
+                <input type='hidden' name='image' t-att-value="acquirer.image_url"/>
```

Comment:
maybe give it a more explicit name than `image`, something like `stripe_image` to avoid conflicts.